RecoLocalCalo/HGCalRecProducers/python/HGCalRecHit_cfi.py

```diff
@@ -114,7 +114,63 @@
 
 
 from Configuration.Eras.Modifier_phase2_hgcalV9_cff import phase2_hgcalV9
-phase2_hgcalV9.toModify( dEdX, weights = dEdX_weights_v9 ) 
+phase2_hgcalV9.toModify( dEdX, weights = dEdX_weights_v9 )
+
+dEdX_weights_v10 = cms.vdouble(0.0,      # there is no layer zero
+                               8.894541,  # Mev
```

Comment:
isn't there a place in the DB for things like this?

Reply:
there is no HGCal DB yet. This has been an outstanding issue for years, and probably will continue to be.

Reply:
- does it mean there is no plan to keep HGCAL conditions data in DB?
- or does it mean that getting data to the DB is so complicated that it's easier to embed all variants of it into the CMSSW release?

Reply:
I'm sure there is a plan to keep HGCal conditions in the DB on the scale of the installation timeline. But in general, at least in my experience, setting up and maintaining DB objects is quite person- and time-intensive. I am not sure there is any existing effort in the DPG devoted to this.

Reply:
@slava77 
it is very clear to the DPG that the most appropriate (only?) way to take care of these calibration constants is via the EventSetup+DB. On the other hand the current situation is still quite fluid and we prefer to have it fixed and available quickly rathter than going through the cumbersome procedure of switching to the DB. Needless to say this switch will come, but I do not have, as of today, a timescale.